src/Symfony/Component/Notifier/Texter.php

```diff
@@ -51,7 +52,11 @@ public function supports(MessageInterface $message): bool
     public function send(MessageInterface $message): ?SentMessage
     {
         if (null === $this->bus) {
-            $this->transport->send($message);
+            $sentMessage = $this->transport->send($message);
```

Comment:
Do we really need this here? It is calling send() method of the transport which already throws the event 🧐